### PR TITLE
haaseduk1 examples

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
Security Check : 81ef62694071b89f544cdfed426ad2cb424ca33e

**Comment by skylarCai** [comment]
auto build success,number:204

**Comment by skylarCai** [comment]
autotest fail,number:1542

**Review by wushengtws — APPROVED** [review]

**Comment by skylarCai** [comment]
Security Check : 81ef62694071b89f544cdfed426ad2cb424ca33e

File: haas_lib_bundles/python/docs/examples/fire_detector/haaseduk1/code/main.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+# 物联网平台连接标志位
+iot_connected = False
+
+# 三元组信息
+productKey = "产品密钥"
+deviceName = "设备名称"
+deviceSecret = "设备密钥"
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/fire_detector/haaseduk1/code/main.py:24]: LGFM

File: haas_lib_bundles/python/docs/examples/floriculture/haaseduk1/code/main.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+# 物联网设备实例
+device = None
+
+# 三元组信息
+productKey = "产品密钥"
+deviceName = "设备名称"
+deviceSecret = "设备密钥"
+
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/floriculture/haaseduk1/code/main.py:29]: LGFM

File: haas_lib_bundles/python/docs/examples/gas_detector/haaseduk1/code/main.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+import ujson                  # json库
+
+adcObj = 0                 # ADC类型的外设对象
+mq2Dev = 0                 # MQ2燃气传感器对象
+alarmLed = 0               # 报警LED对象
+alarmOn = 0                # 记录报警状态
+
+# 物联网平台连接标志位
+iot_connected = False
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/gas_detector/haaseduk1/code/main.py:25]: LGFM

File: haas_lib_bundles/python/docs/examples/hcho_detector/haaseduk1/README.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+<br>
+
+> Python脚本的详细说明请参考脚本内嵌的文字版注释
+
+1. **修改路由器名称及密码**
+
+&emsp;&emsp;
+修改main.py中wifiSsid和wifiPassword的值为读者实际要连接的路由器的名称及密码（请注意名称和密码都需要放在""符号中间）。
+
+```python
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/hcho_detector/haaseduk1/README.md:210]: LGFM

File: haas_lib_bundles/python/docs/examples/hcho_detector/haaseduk1/code/main.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+import ujson                     # json字串解析库
+from driver import ADC           # ADC类，通过微处理器的ADC模块读取ADC通道输入电压
+import hcho                      # 甲醛hcho传感器类
+adcObj = 0                     # ADC通道对象
+uartObj = 0                    # UART通道对象
+
+
+# 物联网平台连接标志位
+iot_connected = False
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/hcho_detector/haaseduk1/code/main.py:15]: LGFM

File: haas_lib_bundles/python/docs/examples/human_detector/haaseduk1/README.md [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+<br>
+
+> Python脚本的详细说明请参考脚本内嵌的文字注释
+
+
+1. **修改路由器名称及密码**
+
+&emsp;&emsp;
+修改main.py中wifiSsid和wifiPassword的值为读者实际要连接的路由器的名称及密码（请注意名称和密码都需要放在""符号中间）。
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/human_detector/haaseduk1/README.md:199]: LGFM

File: haas_lib_bundles/python/docs/examples/human_detector/haaseduk1/code/main.py [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+import ir                        # ir人体红外传感器类
+
+gpioirDev = 0
+gpioledDev = 0
+irDev = 0
+
+# 物联网平台连接标志位
+iot_connected = False
+wlan = None
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/human_detector/haaseduk1/code/main.py:17]: LGFM

File: haas_lib_bundles/python/docs/examples/smart_public_lighting/haaseduk1/README.md [hunk 8]
```diff
@@ -0,0 +1,306 @@
...
+
+&emsp;&emsp;
+之后对代码进行如下修改。
+
+1. **修改路由器名称及密码**
+
+&emsp;&emsp;
+修改main.py中wifiSsid和wifiPassword的值为读者实际要连接的路由器的名称及密码（请注意名称和密码都需要放在""符号中间）。
+
+```python
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/smart_public_lighting/haaseduk1/README.md:178]: LGFM

File: haas_lib_bundles/python/docs/examples/smart_public_lighting/haaseduk1/code/main.py [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+from driver import GPIO
+import netmgr as nm
+import ujson
+
+# 三元组信息
+productKey = "产品密钥"
+deviceName = "设备名称"
+deviceSecret = "设备密钥"
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/smart_public_lighting/haaseduk1/code/main.py:14]: LGFM

File: haas_lib_bundles/python/docs/examples/vehicle_location/haaseduk1/code/main.py [hunk 10]
```diff
@@ -0,0 +1,137 @@
+from aliyunIoT import Device     # aliyunIoT组件是连接阿里云物联网平台的组件
+import utime
+import netmgr as nm
+from driver import UART
+import gnss
+import ujson
+
+uartDev = 0
+gnssDev = 0
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/vehicle_location/haaseduk1/code/main.py:10]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**